### PR TITLE
Add autocomplete, fix mailer, modify google drive config, fixed signup

### PR DESCRIPTION
**Pull request by flynncowell**

### CODE REVIEW

File: api/firebase.js [hunk 1]
```diff
@@ -415,11 +420,12 @@ module.exports = {
 
 	assignEditor: function(toUpdate, id, dept, req, res) {
```
Comment (Niklex21) [inline api/firebase.js:421]: What format is toUpdate['editor']?

File: api/googledrive.js [hunk 2]
```diff
@@ -19,15 +19,11 @@ const TOKEN_PATH = 'credentials.json';
...
-      client_id, client_secret, redirect_uris[0]);
+  const oAuth2Client = new google.auth.OAuth2(client_id, client_secret, redirect_uris[0]);
 
   // Check if we have previously stored a token.
-  fs.readFile(TOKEN_PATH, (err, token) => {
-    if (err) return getAccessToken(oAuth2Client, callback);
-    oAuth2Client.setCredentials(JSON.parse(token));
-    callback(oAuth2Client);
-  });
+  oAuth2Client.setCredentials(JSON.parse(process.env.GOOGLE_DRIVE_CRED));
```
Comment (Niklex21) [inline api/googledrive.js:25]: Seems like the token js file is not used anymore, instead taking the value from an env variable. What if the variable is not set? Is there a fallback function? Also, if the js file is not used, then it should be removed from here.

Comment (flynncowell) [reply api/googledrive.js:25]: the token is autogenerated by Drive, whereas the env var replaced the credentials.json (which is not autogenerated).

Comment (flynncowell) [reply api/googledrive.js:25]: the token is autogenerated by Drive, whereas the env var replaced the credentials.json (which is not autogenerated).

Comment (youngscientists) [reply api/googledrive.js:25]: Did we test this though? I think I tried running this a while ago and it didn’t work...

File: api/firebase.js (hunk 1 above)
Comment (youngscientists) [reply api/firebase.js:421]: Oh it’s just a string in the format “First Last - editor@email.com” sent from the form on the article overview page.

**Pull request merged**